frontend/components/vault/VaultLiquidationTransactionTable.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+    <div class="flex flex-col space-y-1 text-gray-700 dark:text-gray-100">
+        <div class="flex justify-between">
+            <div>Proximity to liquidation threshold</div>
+            <div>
+                <span v-if="vaultTransaction.proximityToLiquidation < 0" class="text-green-500 mr-1">
+                    (Ready for liquidation)
+                </span>
+                <span v-if="vaultTransaction.proximityToLiquidation">
+                    {{ vaultTransaction.proximityToLiquidation }}%
+                </span>
+                <span v-else class="opacity-50"> </span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Next price update</div>
+            <div class="flex items-center space-x-1">
+                <template v-if="vaultTransaction.nextPriceChange">
```

Comment:
Same here and for every other key that is required by the types

Reply:
Same as above.